test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -137,4 +139,11 @@ public static void assertTextFilesEqual(Path path1, Path path2) throws IOExcepti
         List<String> list2 = Files.readAllLines(path2, Charset.defaultCharset());
         assertEquals(String.join("\n", list1), String.join("\n", list2));
     }
+
+    /**
+     * Asserts that the file given does not exist on the filesystem.
+     */
+    public static void assertFileDoesNotExist(String filePath) {
+        assertTrue(Files.notExists(Paths.get(filePath)));
```

Comment:
It's best to put potentially reusable functions in Util classes. That way, when someone needs a similar function in future, they can look through those Util classes to see if it already exists, instead of going through the entire code base. 

As a general rule, any code that is not tightly coupled to the class in which it originated can be generalised and pushed out of the class to a more suitable location. That way, our classes can remain lean and cohesive while maximising reuse.